node/src/chain_spec/oak.rs

```diff
@@ -343,8 +343,7 @@ fn testnet_genesis(
 	general_councils: Vec<AccountId>,
 	technical_memberships: Vec<AccountId>,
 ) -> oak_runtime::GenesisConfig {
-	let candidate_stake =
-		std::cmp::max(oak_runtime::MinCollatorStk::get(), oak_runtime::MinCandidateStk::get());
+	let candidate_stake = oak_runtime::MinCandidateStk::get();
```

Comment:
Yes.

In the Moonbeam parachain-staking crate, MinCollatorStk has been removed, leaving only MinCandidateStk.

<img width="1059" alt="image" src="https://github.com/OAK-Foundation/OAK-blockchain/assets/16951509/7f08c3a5-6967-4592-b015-9f9d9844129f">

Reply:
Good call! @v9n

I sync’ed with Charles. The value of `MinCandidateStk` was updated from 400K to 1M 5 months ago but `MinCollatorStk` wasn’t, so we haven’t been using `MinCollatorStk` for a while. Therefore, since its value has been already out-of-date I think it’s safe to delete its logic now. As Charles said, the `MinCandidateStk` alone can safe guard the candidacy entrance.